.github/workflows/lcm_dreamshaper_cpp.yml

```diff
@@ -138,7 +138,6 @@ jobs:
           . "${{ env.OV_INSTALL_DIR }}/setupvars.ps1"
           python .\samples\python\text2image\main.py .\models\lcm_dreamshaper_v7\FP16 "cyberpunk cityscape like Tokyo New York with tall buildings at dusk golden hour cinematic lighting"
         env:
-          PATH: ${{ env.build_dir }}\openvino_genai
```

Comment:
Shouldn't PATH still get `${{ env.build_dir }}\openvino_genai`? It just needs to be moved to the script to prepended to the old PATH value.

Reply:
but it magically worked w/o it ..